package.json

```diff
@@ -31,13 +31,20 @@
   "homepage": "https://github.com/Devnilson/fisima#readme",
   "devDependencies": {
     "@types/jest": "^25.2.3",
+    "husky": "^4.2.5",
     "jest": "^26.0.1",
-    "prettier": "^2.0.5",
+    "prettier": "2.0.5",
+    "pretty-quick": "^2.0.1",
```

Comment:
what its that?

Reply:
running prettier quickly on staged files so it runs on pre-commit